changeset:   115:3e18fa0cafc4
tag:         tip
user:        "Daniel P. Berrange <berrange@redhat.com>"
date:        Fri Mar 02 09:16:33 2007 -0500
summary:     Fix restarting of guest after install completes

diff -r 5a1190f54c80 -r 3e18fa0cafc4 virt-install
--- a/virt-install	Thu Mar 01 10:24:34 2007 -0500
+++ b/virt-install	Fri Mar 02 09:16:33 2007 -0500
@@ -475,7 +475,7 @@ def main():
             # things installed enough that we should be able to restart
             # the domain
             print "Guest installation complete... restarting guest."
-            dom.start()
+            dom.create()
         else:
             print ("Domain installation does not appear to have been\n"
                    "successful.  If it was, you can restart your domain\n"


